.happy/terraform/modules/schema_migration/main.tf

```diff
@@ -44,15 +44,15 @@ resource aws_batch_job_definition schema_migrations_swap {
     resourceRequirements = [
       {
         type= "VCPU",
-        Value="6"
+        Value="38"
       },
       {
         Type="MEMORY",
-        Value = "47500"
+        Value = "304000"
```

Comment:
don't need to leave a comment, but curious for myself why these numbers? educated estimates based on testing? some calculation?

Reply:
I chose this number because it's the amount of swap that was previously allocated + `MEMORY` that we previously had. I rounded up to 300000 to add more buffer, but migration worked before with it being less, so i'll change it to 250000. It'll save us some compute.